### ISSUE TITLE
[HOLD for payment 2023-08-14] Update App to React native 0.72 version

### ISSUE THREAD

**Issue by mountiny** [Expensify/App#18444]
Let's stay up to date with the latest React native version for the best performance and features available and update our RN version to 0.72

Previous internal issue https://github.com/Expensify/Expensify/issues/275274

**Comment by fabioh8010** [comment]
Hi, I'm Fábio from Callstack - expert contributor group - I would like to work on this issue

**Comment by fabioh8010** [comment]
PR is ready: https://github.com/Expensify/App/pull/18507

**Comment by mountiny** [comment]
Being hyper focused on the EC3 committment so unable to help out here much, hopefully we can get more time on this next week

**Comment by michaelhaxhiu** [comment]
New Feature isn't as pressing as Bug. But we'll get this done soon ⏲️ 🧇

**Comment by mountiny** [comment]
@fabioh8010 continues working on this

**Comment by mountiny** [comment]
Not overdue we changed strategy

**Comment by mountiny** [comment]
@fabioh8010 is focusing on this 100% now, there is a stable release 🎉

**Comment by mountiny** [comment]
Still making progress

**Comment by michaelhaxhiu** [comment]
Last comment on the PR was ~7 hours ago https://github.com/Expensify/App/pull/18507#issuecomment-1630812013

It's progressing!

**Comment by mountiny** [comment]
Still in review

**Comment by mountiny** [comment]
We are 🤏 this close

**Comment by michaelhaxhiu** [comment]
Note: I'm preparing to go OOO for ~2 weeks. @MitchExpensify can I ask you for a favor to ensure payment is handled properly. 

I am not sure how this will get compensated because the PR https://github.com/Expensify/App/pull/18507 has numerous C+ assigned but I think @mountiny has a clear sense of that piece!

**Comment by mountiny** [comment]
I believe its @aimane-chnaif and @parasharrajat being paid $1000 for this PR great job! 

agency C+ are paid differently.

**Comment by MitchExpensify** [comment]
Thanks! And just to be sure @vitHoracek, is it ok to send the payment now and close this out?

**Comment by roryabraham** [comment]
Yes we can close this out. We'll have another follow-up soon, but that's separate

**Comment by MitchExpensify** [comment]
Thanks @roryabraham ! Offer sent [here](https://www.upwork.com/jobs/~010996343f450c5b40) for payment before closure cc @parasharrajat @aimane-chnaif

**Comment by MitchExpensify** [comment]
Paid @aimane-chnaif! 

@parasharrajat Have you made the NewDot request for $1000?

**Comment by suraj-mandal** [comment]
Hi! I found this issue while going through the job description on upwork. Is this issue still existing? If yes, I would like to work on this

**Comment by puneetlath** [comment]
The automation didn't work for some reason. Added hold for payment to the title.

**Comment by JmillsExpensify** [comment]
@MitchExpensify In the future can you summarize payment in one post? I'll accept the above since multiple people confirmed the amount.

**Comment by JmillsExpensify** [comment]
Reviewed the details for @parasharrajat. Approved by payment in NewDot based on the $1,000 BZ summary above.

**Comment by MitchExpensify** [comment]
> @MitchExpensify In the future can you summarize payment in one post? 

For sure!

**Comment by MitchExpensify** [comment]
Closing as we're all paid out here and no BZ steps necessary
